src/Twig/Handler/ArrayHandler.php

```diff
@@ -110,8 +110,8 @@ private function orderHelper($a, $b)
                 return 0;
             }
 
-            $aVal = $a[$this->order_on_secondary];
-            $bVal = $b[$this->order_on_secondary];
+            $aVal = (is_array($a) ? $a[$this->order_on] : $a->{$this->order_on_secondary});
+            $bVal = (is_array($b) ? $b[$this->order_on] : $b->{$this->order_on_secondary});
```

Comment:
Sorry guys... I'm really shredded apart... I will focus and givin a next 
try. Promise... 😀


On 29 April 2016 18:14:29 Bob den Otter <notifications@github.com> wrote:

>> @@ -110,8 +110,8 @@ private function orderHelper($a, $b)
>>                  return 0;
>>              }
>>
>> -            $aVal = $a[$this->order_on_secondary];
>> -            $bVal = $b[$this->order_on_secondary];
>> +            $aVal = (is_array($a) ? $a[$this->order_on] : 
>> $a->{$this->order_on_secondary});
>> +            $bVal = (is_array($b) ? $b[$this->order_on] : 
>> $b->{$this->order_on_secondary});
>
> I think this is still not right. One `order_on` left. :-)
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/bolt/bolt/pull/5257/files/9a692740b54b30f41524977f3e20deee558091c3#r61602975